lib/act.js

```diff
@@ -9,8 +9,9 @@ var call;
 var config;
 var widget;
 var cache;
+var debug = require('debug')('fh-mbaas-api:act');
```

Comment:
Timestamps are included automatically when you do DEBUG_COLORS=0, or it's running on non-TTY terminal. I don't know why they did that. 
I wouldn't include another wrapper if it's not necessary.